<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCallRequestDialog</class>
 <widget class="QWidget" name="VideoCallRequestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoCallRequestDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
background-color: #ffffff;</string>
  </property>
  <widget class="QLabel" name="m_msgLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>18</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>花未眠发起了视频俩天</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_avatarLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../res/res.qrc">:/img/avatar.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_acceptPushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>45</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8fc0e7;</string>
   </property>
   <property name="text">
    <string>接受</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_rejectPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>45</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8fc0e7;</string>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
